includes/quiz-submission/repositories/class-answer-comments-repository.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * File containing the Answer_Comments_Repository class.
+ *
+ * @package sensei
+ */
+
+namespace Sensei\Quiz_Submission\Repositories;
+
+use DateTime;
+use Sensei\Quiz_Submission\Models\Answer;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Answer_Comments_Repository.
+ *
+ * @since $$next-version$$
+ */
+class Answer_Comments_Repository implements Answer_Repository_Interface {
+	/**
+	 * Create a new answer.
+	 *
+	 * @param int    $submission_id The submission ID.
+	 * @param int    $question_id   The question ID.
+	 * @param string $value         The answer value.
+	 *
+	 * @return Answer The answer model.
+	 */
+	public function create( int $submission_id, int $question_id, string $value ): Answer {
+		$comment_answers = $this->get_comment_answers( $submission_id );
+
+		$comment_answers[ $question_id ] = $value;
+
+		$this->update_comment_answers( $submission_id, $comment_answers );
+
+		return new Answer( 0, $submission_id, $question_id, $value, new DateTime() );
```

Comment:
What do you think about creating a child class for the "comments" implementation?
In this implementation, we don't need to pass ID in constructor and we can throw an exception in `get_id()`. (Not sure if there are other differences.)

Reply:
Actually, I think to keep it this way just to keep things simple and avoid having to deprecate the comments model and the interface in the future. In my opinion, it's ok to have some fake/missing data in the model as long as the repository can handle it.